sky/clouds/aws.py

```diff
@@ -289,78 +305,79 @@ def _make(instance_list):
         # `aws` to access private storage buckets.
         # `aws configure list` does not guarantee this file exists.
         if not os.path.isfile(os.path.expanduser('~/.aws/credentials')):
-            return (False, '~/.aws/credentials does not exist.' + help_str)
+            return (False, '~/.aws/credentials does not exist. ' +
+                    self._STATIC_CREDENTIAL_HELP_STR)
 
         # Checks if the AWS CLI is installed properly
         try:
             _run_output('aws configure list')
         except subprocess.CalledProcessError:
             return False, (
                 'AWS CLI is not installed properly.'
-                ' Run the following commands under sky folder:'
-                # TODO(zhwu): after we publish sky to PyPI,
-                # change this to `pip install sky[aws]`
+                ' Run the following commands in the SkyPilot codebase:'
                 '\n     $ pip install .[aws]'
-                '\n   Credentials may also need to be set.' + help_str)
+                '\n   Credentials may also need to be set. ' +
+                self._STATIC_CREDENTIAL_HELP_STR)
 
         # Checks if AWS credentials 1) exist and 2) are valid.
         # https://stackoverflow.com/questions/53548737/verify-aws-credentials-with-boto3
         try:
-            self.get_cloud_user_identity()
+            self.get_current_user_identity()
         except exceptions.CloudUserIdentityError as e:
-            return False, (str(e) + help_str)
+            return False, str(e)
 
         # Fetch the AWS availability zones mapping from ID to name.
         from sky.clouds.service_catalog import aws_catalog  # pylint: disable=import-outside-toplevel,unused-import
         return True, None
 
-    def get_cloud_user_identity(self) -> Optional[str]:
+    def get_current_user_identity(self) -> Optional[str]:
         """Returns the identity of the user on this cloud."""
         try:
             sts = aws.client('sts')
-            # The caller identity contains: UserId, AccountId, Arn.
-            # AWS user ID is unique across all AWS entity. We use it because:
-            # AccountId: can be shared by multiple users under the same
+            # The caller identity contains 3 fields: UserId, AccountId, Arn.
+            # 'UserId' is unique across all AWS entity, which looks like
+            # "AROADBQP57FF2AEXAMPLE:role-session-name"
+            # 'AccountId' can be shared by multiple users under the same
             # organization
-            # Arn: is the full path to the user, which can be reused when
+            # 'Arn' is the full path to the user, which can be reused when
             # the user is deleted and recreated.
+            # Refer to https://docs.aws.amazon.com/cli/latest/reference/sts/get-caller-identity.html # pylint: disable=line-too-long
+            # and https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_identifiers.html # pylint: disable=line-too-long
```

Comment:
Looks like 2nd link can be replaced with https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_variables.html#principaltable?

Reply:
Good point! Replaced. Thanks